Source/WebSocket/hcwebsocket.cpp

```diff
@@ -142,8 +141,7 @@ Result<std::shared_ptr<WebSocket>> WebSocket::Initialize()
     std::shared_ptr<WebSocket> websocket{ new (a.allocate(1)) WebSocket
```

Comment:
I'm pretty sure you have `http_allocate_shared` for this

Reply:
Same reason here - private constructor.